### ISSUE TITLE
[MAINTENANCE] Fix accessibility issues on ror-site and ror-app

### ISSUE THREAD

**Issue by amandafrench** [ror-community/ror-roadmap#335]
**Service/repository**

ROR search app and website

**Describe the current state/issue**

The ROR website and ROR app should be evaluated to see if they meet accessibility standards and issues addressed. 

Some issues on the ROR website have already been identified using [the WAVE browser extension](https://wave.webaim.org/) and fixed: see [report](https://docs.google.com/document/d/1Plsigy49q4juxcQVFVSK_lHHaZvTx_NixxCJNcypb58/edit?usp=sharing). 

Issues remaining on the website relate to color and contrast, and repairing these would entail slight changes to ror-app styles as well. New palette has been approved by ROR director.

WAVE identifies four small issues with the app. 

**Describe the desired state/solution**

- ROR website and app: Use palette https://coolors.co/53baa1-505759-e89005-efefef-e0004d-520c70-293030 
- ROR app: Add form label to input 
- ROR app: Add search button with value
- ROR app: Add language to app pages
- ROR app: Change text color on "Apply" filter button from white to black

**Additional notes**

WAVE report for ROR app
<img width="2392" height="1624" alt="Image of WAVE report" src="https://github.com/user-attachments/assets/af9bc01b-d3c8-491d-a531-d7dcc555fee3" />

Apart from the above four issues on the ROR app, the only other visible change would be that the "Subscribe / Suggest" buttons in the footer would be Raspberry instead of Sinopia red on both the website and the app.

<img width="2528" height="1608" alt="Image of proposed color changes" src="https://github.com/user-attachments/assets/1af7128b-e558-48c3-b903-230487ea735c" />

**Comment by amandafrench** [comment]
Related to https://github.com/ror-community/ror-adoption/issues/98

**Comment by amandafrench** [comment]
Accessibility issues for app resolved with https://github.com/ror-community/ror-app/releases/tag/1.9.10 and accessibility issues for website resolved with https://github.com/ror-community/ror-site/commit/a6febc60c984fbbc5dc4a23792720e2fe6f8433d
